de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/JsonHandler.java

```diff
@@ -36,108 +34,97 @@
 
   protected ScriptHandler scriptHandler;
   protected ExecutionContext exec;
-  
+  protected ParameterData parameterJson;
+  protected ObjectMapper MAPPER = new ObjectMapper();

```

Comment:
Object mappers are expensive, they take time and memory to create one. We could reuse one from FskPlugin.